Makefile

```diff
@@ -45,3 +45,13 @@ fmt:
 .PHONY: lint
 lint:
 	$(rundep) github.com/golangci/golangci-lint/cmd/golangci-lint run --config .github/golangci.yml
+
+.PHONY: imports
```

Comment:
can we merge in `make fmt` instead?

Reply:
Merged:

[7eb92d2](https://github.com/gnolang/gno/pull/1302/commits/7eb92d2b805ac75dd5621de97856894847e46387)


We should think about `goimports` linting for `*.gno` imports in the future